src/utils/xcmApi/index.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  createSubstrateAPI,
+  CollateralUnit,
+  DefaultTransactionAPI
+} from '@interlay/interbtc-api';
+import {
+  Currency,
+  MonetaryAmount
+} from '@interlay/monetary-js';
+import { web3FromAddress } from '@polkadot/extension-dapp';
+import { AddressOrPair } from '@polkadot/api/types';
+
+import {
+  RELAYCHAIN_URL,
+  RELAYCHAIN_ID
+} from '../../constants';
+
+const xcmTransfer = async (
+  // Pass this in explicitly to avoid creating
+  // a dependency on the interBCT api instance
+  originatingAccount: AddressOrPair,
+  destinationAddress: string,
+  transferAmount: MonetaryAmount<Currency<CollateralUnit>, CollateralUnit>
+): Promise<void> => {
+  // Create api and transaction api instances on the parachain
+  const api = await createSubstrateAPI(RELAYCHAIN_URL);
+  const transactionApi = new DefaultTransactionAPI(api, originatingAccount);
```

Comment:
Nice one reusing the the transaction API, not sure if it's compatible with the relay chain tohugh - need to test

Reply:
It worked when I was calling a test transaction from the UI, but I wouldn't call that robust testing :) hopefully will continue to work though!